yt_dlp/extractor/youtube.py

```diff
@@ -6965,22 +6965,29 @@ class YoutubeSearchIE(YoutubeTabBaseInfoExtractor, SearchInfoExtractor):
     IE_DESC = 'YouTube search'
     IE_NAME = 'youtube:search'
     _SEARCH_KEY = 'ytsearch'
-    _SEARCH_PARAMS = 'EgIQAQ%3D%3D'  # Videos only
+    _SEARCH_PARAMS = 'EgIIAfABAQ=='  # Videos only
```

Comment:
Are you sure this is correct? The previous params were % encoded

Reply:
Tested and works. They are encoded in the query anyways